scripts/goimport-file.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
```

Comment:
Maybe this file shouldn't be committed. I just used it once to sort the imports.